AcadeMeData/test_course.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from AcadeMeData.models import Course
+
+
+@pytest.fixture
+def generate_course(generate_degree, generate_professor, course_id=1, name="History of Countries",
+                    elective=True, description="Learn about historic events and their influences on countries"):
+    professor = generate_professor
+    degree = generate_degree
+    course = Course.create_course(course_id=course_id, name=name, degree=degree, elective=elective,
+                                  description=description, professor=professor)
+    course.save()
+    return course
+
+
+@pytest.mark.django_db
+class TestCourseModel:
+    def test_create_course(self, generate_course, name="History of Countries"):
+        course = Course.get_course_by_name(name)
```

Comment:
- seems more like a test for `get_course_by_name` than for `create_course`. 
- where a course is being created here? 
- you usually don't test a fixture.

Reply:
It seems he is preparing the test in fixture, so the test could be renamed to -> get_get_coarse_by_name
generate_course, name="History of Countries"

Reply:
Yes, as I said, it can be a test for 'get_course_by_name`, not for creating a course.

Reply:
Changed name to "test_get_course_by_name". :)